dev-cpp/abseil-cpp/abseil-cpp-20230125.2.ebuild

```diff
@@ -56,13 +56,17 @@ src_prepare() {
 }
 
 src_configure() {
-	local mycmakeargs=(
-		-DCMAKE_CXX_STANDARD=17
-		-DABSL_ENABLE_INSTALL=TRUE
-		-DABSL_LOCAL_GOOGLETEST_DIR="${WORKDIR}/googletest-${GTEST_COMMIT#v}"
-		-DABSL_PROPAGATE_CXX_STD=TRUE
-		-DABSL_BUILD_TESTING=$(usex test ON OFF)
-		$(usex test -DBUILD_TESTING=ON '') #intentional usex, it used both variables for tests.
-	)
-	cmake_src_configure
+	abi_configure() {
+		local mycmakeargs=(
+			-DCMAKE_CXX_STANDARD=17
+			-DABSL_ENABLE_INSTALL=TRUE
+			-DABSL_LOCAL_GOOGLETEST_DIR="${WORKDIR}/googletest-${GTEST_COMMIT#v}"
+			-DABSL_PROPAGATE_CXX_STD=TRUE
+			-DABSL_BUILD_TESTING=$(usex test ON OFF)
+			$(usex test -DBUILD_TESTING=ON '') #intentional usex, it used both variables for tests.
+		)
+		cmake_src_configure
+	}
+
+	multilib_parallel_foreach_abi abi_configure
```

Comment:
`multilib_parallel_foreach_abi` is deprecated, but could you try defining `multilib_src_configure` instead, then we don't need the nesting? (hoist abi_configure into top-level, rename to multilib_src_configure).